crates/db_schema/src/source/community.rs

```diff
@@ -33,15 +33,15 @@ pub struct Community {
   pub inbox_url: DbUrl,
   #[serde(skip)]
   pub shared_inbox_url: Option<DbUrl>,
+  pub hidden: bool,
+  pub posting_restricted_to_mods: bool,
+  pub instance_id: InstanceId,
   /// Url where moderators collection is served over Activitypub
```

Comment:
The correct ordering of columns is necessary for diesel.

Reply:
Then its better to mention this in a comment in the code.

Reply:
Its something that's required for every table, and a part of diesel, I just wanted to note it in case you were confused why I moved some variables down a few lines.